full/src/test/java/apoc/load/LoadDirectoryTest.java

```diff
@@ -48,6 +50,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.neo4j.test.assertion.Assert.assertEventually;
 
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
is that required?

Reply:
I had to put them in order to find the one(s) that caused the first error